### PR TITLE
Unit test v1

### PR DESCRIPTION
**Pull request by mersanuzun**

### CODE REVIEW

**Comment by mersanuzun** [comment]
AppBanner unit test is broken. I fixed it and added 3 more component tests, fyi @fatihacet

**Comment by fatihacet** [comment]
@abdullah would you mind reviewing this one too since you already did the first one?

@mersanuzun thanks for adding all those specs, great work 👍

**Comment by mersanuzun** [comment]
@abdullah I decided to use `RouterLinkStub` from vue-test-utils. Now, `to` prop is covered.

Image src was tested using snapshot.

File: tests/unit/src/components/shared/JobItem.spec.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    expect(companyLink.props('to')).toEqual('/firmalar/TEST_SLUG');
+  });
+
+  it('should be correct location url icons for company', () => {
+    const companyLocationLink = wrapper.find('ul.listing-icons li:nth-child(2) a');
+
+    expect(companyLocationLink.props('to')).toEqual(`/ilan-ara?location=${post.location}`);
+  });
+
+  it('should be correct ur for company slug', () => {
```
Comment (abdullah) [inline tests/unit/src/components/shared/JobItem.spec.js]: Spelling mistake. Should it be `url`?

**Comment by abdullah** [comment]
@mersanuzun There is one spelling mistake. Can you fix it. After that, I will merge it ASAP.

**Comment by mersanuzun** [comment]
> @mersanuzun There is one spelling mistake. Can you fix it. After that, I will merge it ASAP.

@abdullah  Done :)

**Comment by fatihacet** [comment]
@abdullah feel free to merge when you are ok with it but please use Squash and Merge.

**Pull request merged**